tracing/tracing.go

```diff
@@ -33,9 +39,32 @@ func defaultConfig() *config {
 		serviceName: envOrDefault(signalfxServiceName),
 		accessToken: envOrDefault(signalfxAccessToken),
 		url:         envOrDefault(signalfxEndpointURL),
+		globalTags:  envGlobalTags(),
+
 	}
 }
 
+// envGlobalTags extract global tags from the environment variable and parses the value in the expected format
+// key1:value1,
+func envGlobalTags() []tracer.StartOption {
+	var globalTags []tracer.StartOption
+	if val := os.Getenv(signalfxSpanTags); val != "" {
+		tags := strings.Split(val, ",")
+		for _, tag := range tags {
+			pair :=strings.Split(tag, ":")
```

Comment:
any need to handle strings with multiple `:` if yes I would suggest doing it as a separate PR, if not document that currently they are not supported. These are very common (ipv6, URLs, etc) I suspect that we need to support them, commas seem less like to be required.

Reply:
This is a good point - I will update the documentation and create a follow up story to handle multiple `:`